core/capabilities/webapi/target/capability.go

```diff
@@ -0,0 +1,168 @@
+package target
```

Comment:
I remember seeing a discussion about naming and file structure, but I find capabilities/webapi/target/capability.go file exporting package target and tested by target_test.go to be a bit confusing.  I'd prefer the filename to be target.go, even though that's redundant.  But maybe there's general guidance against duplicating folder and filename.

Reply:
➕

Reply:
Will make this change.